app/partials/actions/add.html

```diff
@@ -1,10 +1,10 @@
 <div class="container">
 	<h3>
-		Add {{rdesc.name.singular}}
+		Add {{rdesc.attributes.name.singular}}
 	</h3>
-	<h5>{{rdesc.description}}</h5>
+	<h5>{{rdesc.attributes.description}}</h5>
 
-	<div class="form-group" ng-repeat="field in rdesc.fields" ng-include="'/partials/fields/default-field.html'">
+	<div class="form-group" ng-repeat="field in rdesc.attributes.fields" ng-include="'/partials/fields/default-field.html'">
```

Comment:
So, I think the API documentation is going to need to distinguish between fields that _clients can't even set on create_, like facebookId, and fields that clients can set initially but that then can't be modified afterwards. Do you have a preferred naming scheme to distinguish between these two cases (maybe `readOnly` vs. `unmodifiable`?) Either way, right now `readOnly` describes the first case, which means that `readOnly` fields shouldn't even be included in the add form, and so should be filtered out in the ng-repeat (which I'm assuming is easy, but idk)